paasta_tools/kubernetes_tools.py

```diff
@@ -1791,8 +1803,18 @@ def get_pod_template_spec(
             "paasta.yelp.com/service": self.get_service(),
             "paasta.yelp.com/instance": self.get_instance(),
             "paasta.yelp.com/git_sha": git_sha,
+            "paasta.yelp.com/autoscaled": str(self.is_autoscaling_enabled()).lower(),
+            "paasta.yelp.com/cluster": self.get_cluster(),
+            "paasta.yelp.com/pool": self.get_pool(),
+            # ideally, we'd also have the docker image here for ease-of-use
+            # in Prometheus relabeling, but that information is over the
+            # character limit for k8s labels (63 chars)
+            "paasta.yelp.com/deploy_group": self.get_deploy_group(),
         }
 
+        if self.get_team():
```

Comment:
is the label being absent better than something like `Unknown`?

Reply:
hmm, i can check with dpopes tomorrow to see how OPA handles a value that doesn't map to a team

Reply:
OPA doesn't read this at all, although dpopes mentioned he's worried about confusing people with the `team` construct in OPA and this label (which can also be used in OPA if desired)

that said, i went ahead and made this a default label with a fallback value of Unknown